### ISSUE TITLE
MY 펀딩 UI 레이아웃

### ISSUE THREAD

**Issue by j8won** [Step3-kakao-tech-campus/Team13_FE#56]
##  Description
![image](https://github.com/Step3-kakao-tech-campus/Team13_FE/assets/75734220/00ca132e-0241-427b-bc49-ddef4d50695c)
- 반응형을 포함한 스타일링
- 탭 버튼에 따라 하단 내용 변경

## TO DO
- [ ] 프로필 사진
- [ ] 닉네임
- [ ] 탭 버튼

**Comment by Klomachenko** [comment]
탭 버튼은 상단의 `My 펀딩`과 `내 계정`을 말씀하신거죠?

**Comment by j8won** [comment]
@Klomachenko 아니요! 그건 이미 나비게이션 바에 구현이 되어 있습니다. 탭 버튼은 팔로잉, 찜한 목록, 펀딩 내역, ...의 버튼을 의미합니다. Tabs.jsx를 활용할 예정이에요~

**Comment by j8won** [comment]
근데 이 작업보다 msw 작업을 먼저 해볼까 생각 중이에요 ㅎㅎ...
